editor/editor_help_search.cpp

```diff
@@ -845,6 +889,12 @@ TreeItem *EditorHelpSearch::Runner::_create_class_item(TreeItem *p_parent, const
 		item->clear_custom_color(1);
 	}
 
+	if (p_matching_keyword.is_empty()) {
+		item->set_text(0, p_doc->name);
+	} else {
+		item->set_text(0, p_doc->name + "      (" + TTR("Matches") + ": \"" + p_matching_keyword + "\")");
```

Comment:
You're free to use `vformat` here. Translators have to comply with it.
```suggestion
		item->set_text(0, p_doc->name + "      - " + TTR(vformat("Matches the %s keyword.", p_matching_keyword));
```